src/identity/tezos/TezosLedgerConnector.ts

```diff
@@ -0,0 +1,110 @@
+import Transport from "@ledgerhq/hw-transport";
+import * as bip32path from 'bip32-path';
+
+export enum Curve { // taken from https://github.com/obsidiansystems/ledger-app-tezos/blob/master/APDUs.md
+    ED25519 = 0x00,
+    SECP256K1 = 0x01,
+    SECP256R1 = 0x02
+};
+
+export enum Instruction { // taken from https://github.com/obsidiansystems/ledger-app-tezos/blob/master/APDUs.md
+    INS_VERSION = 0x00,
+    INS_GET_PUBLIC_KEY = 0x02,
+    INS_PROMPT_PUBLIC_KEY = 0x03,
+    INS_SIGN = 0x04,
+    INS_SIGN_UNSAFE = 0x05
+}
+
+export default class TezosLedgerConnector {
```

Comment:
Can we add a comment with the overall purpose of this class?

Reply:
this gets broken out into a module soon